internal/ci/gcp-deploy.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -ex
+
+export GOOGLE_APPLICATION_CREDENTIALS=/keyconfig.json
+deployment=$1
```

Comment:
Can we return an error code of 1 (`exit 1`) if there is no parameter? Or perhaps default to `latest`, although I prefer the former approach.

```bash
if [ "$#" -ne 1 ]; then
    echo "Illegal number of parameters"
    exit 1
fi
```